packages/nodejs/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+- `createIdentityRequest`, to create identity requests.
```

Comment:
I think we should update the CHANGELOG as described here: https://github.com/Concordium/concordium-node-sdk-js/blob/7b8d693cc7e0b8daa765efc0dd96d81c9ba1c73d/README.md instead. Then we can avoid duplicating the CHANGELOG on every update to common.